buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/DetermineBaselines.kt

```diff
@@ -53,6 +58,9 @@ open class DetermineBaselines : DefaultTask() {
     private
     fun Property<String>.isDefaultValue() = !isPresent || get() in listOf("", defaultBaseline, Config.baseLineList)
 
+    private
+    fun currentCommitBaseline() = commitBaseline(project.execAndGetStdout("git", "rev-parse", "HEAD"))
```

Comment:
You could make this a `val` with `get`:
```
val currentCommitBaseline
    get() = commitBaseline(project.execAndGetStdout("git", "rev-parse", "HEAD"))
```

Then you could reference it as a value instead of a method.